draft-ietf-quic-transport.md

```diff
@@ -511,9 +524,9 @@ other fields.
 
 | Type | Packet Number Size |
 |:-----|:-------------------|
-| 0x1F | 1 octet            |
-| 0x1E | 2 octets           |
-| 0x1D | 4 octets           |
+| 0x7  | 1 octet            |
+| 0x6  | 2 octets           |
+| 0x5  | 4 octets           |
```

Comment:
Technically, the reason we were counting back instead of forward was to assist in demuxing.  If we're changing these again anyway, do we still need this unintuitive property?

Reply:
I’m not sure if we still need it, i don’t think so. @csperkins?

Reply:
Let's start counting from 0.  As it is, this arrangement collides with DTLS, but that's hard to avoid.

Reply:
If the plan is to go back to the current types after Google has moved to iQUIC, it really doesn't matter. So I don't care, but leaving this unintuitive property in may be a good thing if that's the long term outcome.

Reply:
If we count from zero, and later set Google QUIC Demultipexing bit back to zero, then we collide with STUN for short header packets with a Connection ID present and key phase zero. Counting down avoids that collision.

If we're going to support P2P QUIC, then having a long-term stable way of easily demuxing with STUN seems important, so I'd prefer to avoid that collision.

Reply:
@csperkins, the current pattern is 0xx10xxx.  That never collides with STUN.  (The omission of bit 0x10 from the PR is a little confusing.)

Reply:
sure - I was responding to the suggestion that the gQUIC demux bits be removed before IETF QUIC goes final, in which case we would have a collision.